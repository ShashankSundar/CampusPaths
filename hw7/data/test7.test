r
66
85
q