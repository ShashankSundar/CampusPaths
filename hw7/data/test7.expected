First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Colonie Apartments to Blitman Residence Commons:
	Walk SouthWest to (Intersection 114)
	Walk South to (Intersection 113)
	Walk West to (Intersection 111)
	Walk West to (Intersection 117)
	Walk West to (Intersection 112)
	Walk SouthWest to (2021 Peoples Avenue)
	Walk West to (Beman Park Firehouse)
	Walk West to (Alumni House)
	Walk SouthWest to (H Building)
	Walk South to (North Hall)
	Walk SouthWest to (Intersection 132)
	Walk West to (Intersection 131)
	Walk West to (Intersection 130)
	Walk SouthWest to (Pittsburgh Building)
	Walk West to (Intersection 94)
	Walk SouthWest to (Winslow Building)
	Walk South to (Louis Rubin Memorial Approach)
	Walk West to (Blitman Residence Commons)
Total distance: 1998.427 pixel units.
